<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="mainTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="mainTab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QRDTTableView" name="msgTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRDTTableView" name="regTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device(&amp;D)</string>
    </property>
    <addaction name="actionInit"/>
    <addaction name="actionPCIE_Restart"/>
    <addaction name="separator"/>
    <addaction name="actionTuneOCXO"/>
    <addaction name="separator"/>
    <addaction name="actionCXU_Restart"/>
    <addaction name="actionCXU_Shutdown"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRFU">
    <property name="title">
     <string>RFU(&amp;R)</string>
    </property>
    <addaction name="actionRFU0"/>
    <addaction name="actionRFU1"/>
    <addaction name="actionRFU2"/>
    <addaction name="actionRFU3"/>
    <addaction name="actionRFU4"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Function(&amp;F)</string>
    </property>
    <addaction name="actionArbViewer"/>
    <addaction name="separator"/>
    <addaction name="actionStarAllIQ_Capture"/>
    <addaction name="actionStopAllIQ_Capture"/>
    <addaction name="separator"/>
    <addaction name="actionLogConfig"/>
    <addaction name="actionLogViewer"/>
    <addaction name="separator"/>
    <addaction name="actionInstrumtControl"/>
    <addaction name="actionCalculator"/>
    <addaction name="actionNI_IO_Trace"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test(&amp;T)</string>
    </property>
    <addaction name="actionProgramFPGAbit"/>
    <addaction name="actionAlgorithm_5355"/>
    <addaction name="actionFittingInterpolation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuRFU"/>
   <addaction name="menuFunction"/>
   <addaction name="menuTest"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInit">
   <property name="text">
    <string>Initialization(&amp;I)</string>
   </property>
   <property name="statusTip">
    <string>Init Device in Menu RFU</string>
   </property>
  </action>
  <action name="actionPCIE_Restart">
   <property name="text">
    <string>PCIE Restart(&amp;P)</string>
   </property>
  </action>
  <action name="actionTuneOCXO">
   <property name="text">
    <string>Tune OCXO(&amp;T)</string>
   </property>
  </action>
  <action name="actionCXU_Restart">
   <property name="text">
    <string>CXU Restart(&amp;R)</string>
   </property>
  </action>
  <action name="actionCXU_Shutdown">
   <property name="text">
    <string>CXU Shutdown(&amp;S)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRFU0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RFU0(&amp;0)</string>
   </property>
  </action>
  <action name="actionRFU1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RFU1(&amp;1)</string>
   </property>
  </action>
  <action name="actionRFU2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RFU2(&amp;2)</string>
   </property>
  </action>
  <action name="actionRFU3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RFU3(&amp;3)</string>
   </property>
  </action>
  <action name="actionRFU4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RFU4(&amp;4)</string>
   </property>
  </action>
  <action name="actionArbViewer">
   <property name="text">
    <string>Arb Viewer(&amp;A)</string>
   </property>
  </action>
  <action name="actionStarAllIQ_Capture">
   <property name="text">
    <string>Start All IQ Capture(&amp;R)</string>
   </property>
  </action>
  <action name="actionStopAllIQ_Capture">
   <property name="text">
    <string>Stop All IQ Capture(&amp;O)</string>
   </property>
  </action>
  <action name="actionLogConfig">
   <property name="text">
    <string>Log Config(&amp;L)</string>
   </property>
  </action>
  <action name="actionLogViewer">
   <property name="text">
    <string>Log Viewer(&amp;V)</string>
   </property>
  </action>
  <action name="actionInstrumtControl">
   <property name="text">
    <string>Instrument Control(&amp;I)</string>
   </property>
  </action>
  <action name="actionCalculator">
   <property name="text">
    <string>Calculator(&amp;C)</string>
   </property>
  </action>
  <action name="actionNI_IO_Trace">
   <property name="text">
    <string>NI IO Trace(&amp;N)</string>
   </property>
  </action>
  <action name="actionProgramFPGAbit">
   <property name="text">
    <string>Program FPGA bit(&amp;P)</string>
   </property>
  </action>
  <action name="actionAlgorithm_5355">
   <property name="text">
    <string>Algorithm 5355(&amp;A)</string>
   </property>
  </action>
  <action name="actionFittingInterpolation">
   <property name="text">
    <string>Fitting &amp; Interpolation(&amp;F)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QRDTTableView</class>
   <extends>QTableView</extends>
   <header>qrdttableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
